src/components/author-primitives/vault/columns/docs.mdx

```diff
@@ -0,0 +1,20 @@
+---
+componentName: 'Columns'
+---
+
+> Note: this component was added ~ 2 years ago for a one-off use case.
+
+An _experimental_ component currently used only on `/api-docs/plugins-directory` -- currently only works correctly when applied to `ul` elements.
```

Comment:
Intended for use in `swingset`, but currently not rendering since our components folder isn't flat.